1.0/docs/devguide/local-dom.md

```diff
@@ -234,14 +265,193 @@ Sometimes it's necessary to access the elements which have been distributed to a
+
+    <simple-carousel>
+      <img src="one.jpg">
+      <img src="two.jpg">
+      <img src="three.jpg">
+    <simple-carousel>
+
+The carousel adds dots underneath the current image that let the user select a different image, so the carousel needs to know how many children it has. This is simple enough: the carousel can check its children the `attached` method:
+
+    attached: function() {
+      this.childCount = Polymer.dom(this).children.length;
+      // do something with childCount ...
+    }
+
+But there are a few issues here. What if you create a new element, `<popup-carousel>`, that includes a simple carousel in its local DOM? You use the new element the same way:
+
+    <popup-carousel>
+      <img src="one.jpg">
+      <img src="two.jpg">
+    </popup-carousel>
+
+Internally, the popup-carousel does something like this:
+
+    <dom-module id="popup-carousel">
+      <template>
+        <simple-carousel>
+          <content></content>
+        </simple-carousel>
+      </template>
+      ...
+    </dom-module>
+
+The popup carousel simply passes its children on to the simple carousel by including a `<content>` tag. But now the simple carousel's `attached` method doesn't work: `Polymer.dom(this).children.length` will always return 1, because the carousel only has a single child, the `<content>` tag. 
+
+Clearly, `children` isn't what you want here. You want a list of children, with any `<content>` tags replaced by their distributed children. Unfortunately, the platform doesn't have a primitive for this, so Polymer has added the concept of "effective children" in its DOM API.
+
+You can retrieve an element's effective child nodes using:
+
+    var effectiveChildren = Polymer.dom(element).getEffectiveChildNodes();
+
+For convenience, several utility methods are available on the Polymer element prototype:
+
+*   `getEffeciveChildNodes()`. Returns a list of effective child nodes for 
+    this element.
+*   `getEffectiveChildren()`. Returns a list of effective child elements for
+    this element.
+*   `queryEffectiveChildren(selector)`. Returns the first effective child 
+    that matches <var>selector</var>. 
+*   `queryAllEffectiveChildren(selector)`. Returns a list of effective
+    children that match <var>selector</var>.
+
+Replacing `children` with the `getEffectiveChildren` method gives you the result you want:
+
+    this.childCount = this.getEffectiveChildren().length;
+
+You can think of `getEffectiveChildren` as a composition-friendly version of `children`.
+
+### Observe added and removed children {#observe-nodes}
+
+Use the DOM API's `observeNodes` method to track when children are added and removed from
+your element:
+
+    this._observer = Polymer.dom(this.$.contentNode).observeNodes(function(info) {
+      this.processNewNodes(info.addedNodes);
+      this.processRemovedNodes(info.removedNodes);
+    });
+
+You pass `observeNodes` callback be invoked when nodes are added or removed.
```

Comment:
That ain't quite English, is it? How about, "You pass `observeNodes` **a** callback **to be** invoked..."?